### ISSUE TITLE
I fell into the great forge and can't get out

### ISSUE THREAD

**Issue by Btataz** [dalaranwow/dalaran-wow#5193]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

I was in Iron forge, and I know this is stupid, but I tried jumping into the great forge. I died almost instantly and can't get out. can't get my ghost out and can't get any help

**Expected behaviour**: Tell us what should happen instead

You could set up an invisible wall or rez people outside it, just don't make it isolated

**Steps to reproduce the problem**:

1. go to the great forge
2. jump into it
3. die

**Include proofs for this behaviour**
![screenshot 22](https://user-images.githubusercontent.com/34771992/34299826-b72b3440-e72d-11e7-931e-eeac1707a91d.png)
![Uploading Screenshot (23).png…]()
![Uploading Screenshot (24).png…]()

https://www.gamefaqs.com/boards/534914-world-of-warcraft/44151244

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Btataz,
Use an in-game ticket system to get help, bugtracker is not for these kind of issues.
